### PR TITLE
fix ingress pr detection; add deployment probes

### PR DESCRIPTION
**Pull request by maflaven**

### CODE REVIEW

File: packs/python-model-microservice/charts/templates/ingress-custom.yaml [hunk 1]
```diff
@@ -13,10 +13,10 @@ metadata:
 spec:
   tls:
     - hosts:
-      - {{ template "custom_hostname3" . }}
+      - {{ template "custom_hostname4" . }}
       secretName: arturo-tls
   rules:
-    - host: {{ template "custom_hostname3" . }}
+    - host: {{ template "custom_hostname4" . }}
```
Comment (yyolk) [inline packs/python-model-microservice/charts/templates/ingress-custom.yaml]: oh no!

File: packs/python-microservice/charts/values.yaml [hunk 2]
```diff
@@ -10,13 +10,25 @@ image:
...
+    failureThreshold: 3
 resources:
   limits:
     cpu: 2
-    memory: 8Gi
+    memory: 8G
   requests:
     cpu: 1
-    memory: 8Gi
+    memory: 8G
```
Comment (yyolk) [inline packs/python-microservice/charts/values.yaml]: thank you

File: packs/python-microservice/charts/values.yaml [hunk 3]
```diff
@@ -10,13 +10,25 @@ image:
...
   name: REPLACE_ME_APP_NAME
   internalPort: 8000
+  livenessProbe:
+    path: /_mgmt/health
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 3
+  readinessProbe:
+    path: /_mgmt/health
```
Comment (yyolk) [inline packs/python-microservice/charts/values.yaml]: did we want this `/health` or `/ping`?

would it be safe to assume `/health` is a `/ping` until its more robust?

Comment (maflaven) [reply packs/python-microservice/charts/values.yaml]: Yes to the latter. Most model microservices have `/health` and `/info` defined—not `/ping`. I don't want this change to break projects' deployments.

**Pull request merged**